python/tests/integration/api40/test_methods.py

```diff
@@ -73,6 +78,62 @@ def test_crud_user(sdk: mtds.Looker40SDK):
     assert resp == ""
 
 
+def test_crud_user_dict(sdk):  # no typing
+    """Test creating, retrieving, updating and deleting a user."""
+
+    # Create user
+    new_user = sdk.create_user(
+        dict(
+            first_name="Rudolphontronix",
+            last_name="Doe",
+            is_disabled=False,
+            locale="fr",
+        )
+    )
+    assert new_user["first_name"] == "Rudolphontronix"
```

Comment:
ok, ok. really .. can't these values be variables defined once? (`test_first_name` and `test_last_name` perhaps?)

Reply:
fair enough, it's just copy/pasta from the `test_crud_user` test but I can remove all these magic strings

Reply:
Thanks!